### PR TITLE
KIRA Featured Artist announcement

### PR DESCRIPTION
**Pull request by Ephemeralis**

### CODE REVIEW

File: news/2018-09-12-new-featured-artist-kira.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+date: 2018-09-12 15:00:00 +0000
+---
+
+Showcasing the best fusion of pop music and Vocaloids to date, we're proud to present our newest addition to the Featured Artist roster - **KIRA**!
+
+[![](https://assets.ppy.sh/artists/27/header.jpg)](https://osu.ppy.sh/beatmaps/artists/27)
+
+Inspired by osu! to start delving into the scene himself, **KIRA** has taken the Vocaloid music community by storm in recent years with several smash pop hits, such as *Circles* and *MONSTER*, the latter netting over *4.3million views on YouTube.* 
+
+Damn.
```
Comment (TPGPL) [inline news/2018-09-12-new-featured-artist-kira.md]: wouldn't it be better to add it at the end of L11?

File: news/2018-09-12-new-featured-artist-kira.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+We know that osu! players by far and large have a definite craving for Vocaloid-themed tracks, and we also know that you guys are partial to a little bit of pop here and there - so we're especially proud to announce that we have **ten** new poppy wonders available for mapping now over at [**KIRA**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/27).
```
Comment (TPGPL) [inline news/2018-09-12-new-featured-artist-kira.md]: We all know

File: news/2018-09-12-new-featured-artist-kira.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Not simply content with being simply a delight to the ears, **KIRA**'s work is also accompanied by a unique music video for each track - all included by default in each download!
+
+Check out a small sample of what's available:
+
+### Monster
```
Comment (TPGPL) [inline news/2018-09-12-new-featured-artist-kira.md]: it's MONSTER iirc

File: news/2018-09-12-new-featured-artist-kira.md (hunk 1 above)
Comment (peppy) [reply news/2018-09-12-new-featured-artist-kira.md]: i think it's fine

**Pull request merged**